notes/handbook.sop.weekly-planning.md

```diff
@@ -32,6 +40,21 @@ This describes Dendron's weekly planning process. Think of it as sprint retrospe
 - [[Everyone|dendron://dendron.handbook/role.everyone]]
   - make any necessary updates in your [[weekly Journal|handbook.sop.weekly-journal]]
 
+### Example
+1. [[#weekly-goals]]
+
+- [[user.bindhu.weekly.2022.06.21]]
+```goal
+## Current Goals
+- [ ] [[Add Process for All Weekly Goals to Be a Task|dendron://private/task.2022.06.15.add-process-for-all-weekly-goals-to-be-a-task]]
```

Comment:
note the differences:

 - its clear whats a file name vs file contents
 - code blocks are marked as markdown blocks
 - we have `...` to show context that there is more or less text

Reply:
high level - we're optimizing for the **reader** experience. it might be easy to c/p a wikiink but this will be read a lot more so make sure this information is easy to consume